docs/api.md

```diff
@@ -142,6 +142,45 @@ devices.
 
 **NOTE:** These are subject to change.
 
+#### raid
+
+This field contains the information about the RAID configuration for bare 
+metal servers. 
+
+The sub-fields are
+
+* *hardwareRAIDVolumes* -- It contains the list of logical disks for hardware 
+  RAID, if rootDeviceHints isn't used, first volume is root volume. Furthermore,
+  it defines the desired configuration of volume in hardware RAID.
+  * *level* -- 'RAID level for the logical disk. The following levels are 
+    supported: 0;1;2;5;6;1+0;5+0;6+0.'
+  * *name* -- Name of the volume. Should be unique within the Node. If not 
+    specified, volume name will be auto-generated.
+  * *numberOfPhysicalDisks* -- Integer, number of physical disks to use for the 
+    logical disk. Defaults to minimum number of disks required for the 
+    particular RAID level.
+  * *rotational* -- Select disks with only rotational or solid-state storage.
```

Comment:
Something like "If true, select only rotational disks, if false - only solid-state and NVMe. Any disk types are used by default."